packages/router/__tests__/router-test.ts

```diff
@@ -14673,18 +14706,11 @@ describe("a router", () => {
         expect(e).toMatchInlineSnapshot(`[Error: queryRoute() call aborted]`);
       });
 
-      it("should require a signal on the request", async () => {
+      it("should assign signals to requests by default (per the spec)", async () => {
         let { queryRoute } = createStaticHandler(SSR_ROUTES);
         let request = createRequest("/", { signal: undefined });
-        let e;
-        try {
-          await queryRoute(request, { routeId: "index" });
-        } catch (_e) {
-          e = _e;
-        }
-        expect(e).toMatchInlineSnapshot(
-          `[Error: query()/queryRoute() requests must contain an AbortController signal]`
```

Comment:
We've since fixed our `Request` implementation to match the spec so it always has a signal.  It's no longer possible to create one without a signal so altered these tests to just confirm that